### PR TITLE
Attempt to fix random robot window test failures

### LINKED ISSUE #4632: CI test suite failure due to robot window test
Very often the CI test suite on Linux is failing due to errors when running the robot window test:
 ```
FAILURE with /home/runner/work/webots/webots/tests/api/worlds/robot_window_html.wbt: Timeout the results file has not been written quickly enough
```

### PR DESCRIPTION
**Pull request by omichel**
Fixes #4632.

The robot windows CI test requires to start a browser on the CI machine. This may be a long process as the browser start-up on these machines may be slow. Maybe it needs to install some updates from time to time.

### CODE REVIEW

File: tests/default/controllers/test_suite_supervisor/test_suite_supervisor.py [hunk 1]
```diff
@@ -239,8 +239,10 @@ def run(self):
             receiver = self.getDevice("ts_receiver")
             receiver.enable(basicTimeStep)
 
-        # 30 seconds before executing the next world
-        timeout = time.time() + 30
+        # 30 seconds before executing the next world, 60 seconds for the robot_window_html test
+        delay = 60 if self.currentSimulationFilename.endswith('/robot_window_html.wbt') else 30
+        print(f'delay = {delay}')
```
Comment (ygoumaz) [inline tests/default/controllers/test_suite_supervisor/test_suite_supervisor.py:244]: Is the printing needed or only for debugging?

**Review by ygoumaz — COMMENTED** [review]
Thank you. Just a minor question.

File: tests/default/controllers/test_suite_supervisor/test_suite_supervisor.py (hunk 1 above)
Comment (omichel) [reply tests/default/controllers/test_suite_supervisor/test_suite_supervisor.py:244]: Oops, yes, I should remove it...

**Review by ygoumaz — APPROVED** [review]
Thank you.

**Pull request merged**